middleware/isLoggedIn.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next) => {
+  const token = req.authorization;
+  req.isAuth = false;
+  if (!req.authorization) {
+    return next();
+  }
+  let decodedToken;
+  try {
+    decodedToken = jwt.verify(token, 'somesupersecretkey', (decodedToken) => {});
```

Comment:
`somesupersecretkey` should be in environment variables

Reply:
yeah this is something i had from when i made the research, do you want me to remove it and add it later? if not i'll let it be there till we do login